docs/resources/elasticsearch_script.md

```diff
@@ -0,0 +1,65 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "elasticstack_elasticsearch_script Resource - terraform-provider-elasticstack"
+subcategory: ""
```

Comment:
`subcategory: "Cluster"` (I think, not sure if this warrants it's own category)

Might need to add a [template](https://github.com/elastic/terraform-provider-elasticstack/tree/master/templates/resources)

Reply:
Oh totally missed it, will fix.
I want this😂
https://github.com/hashicorp/terraform-plugin-docs/issues/156

Reply:
Nice, yep very much want that one too :)